compose/ui/ui/src/androidInstrumentedTest/kotlin/androidx/compose/ui/input/pointer/TestUtils.kt

```diff
@@ -591,22 +592,25 @@ internal class PointerInputChangeEmitter(id: Int = 0) {
     fun nextChange(
         position: Offset = Offset.Zero,
         down: Boolean = true,
-        time: Long = 0
+        time: Long = 0,
+        pointerType: PointerType = PointerType.Touch
     ): PointerInputChange {
         return PointerInputChange(
-            id = pointerId,
-            time,
-            position,
-            down,
-            previousTime,
-            previousPosition,
-            previousPressed,
-            isInitiallyConsumed = false
-        ).also {
-            previousTime = time
-            previousPosition = position
-            previousPressed = down
-        }
+                id = pointerId,
+                time,
+                position,
+                down,
+                previousTime,
+                previousPosition,
+                previousPressed,
+                isInitiallyConsumed = false,
+                type = pointerType
+            )
+            .also {
+                previousTime = time
+                previousPosition = position
+                previousPressed = down
+            }
```

Comment:
This looks indented once too much.

Reply:
this were the rules of new linter in the aosp repo so I just had to have this